docs/resources/excel-samples.md

```diff
@@ -132,6 +132,10 @@ async function main(context: Excel.RequestContext) {
 }
 ```
 
+## Scenario samples
+
+For samples showcasing larger, real-world solutions, visit [Sample Scenarios for Office Scripts](scenarios/sample-scenario-overview.md).
```

Comment:
Update casing in article title:

Sample Scenarios for Office Scripts -> Sample scenarios for Office Scripts